docs/Debugging.md

```diff
@@ -51,6 +51,7 @@ It's possible to debug WebView contents in the iOS simulator or on a device usin
 
 #### Steps:
 
+1. Mark the WebView as being "inspectable" using the [`webviewDebuggingEnabled`](https://github.com/react-native-webview/react-native-webview/blob/master/docs/Reference.md#webviewDebuggingEnabled) prop
 1. Open Safari Preferences -> "Advanced" tab -> enable checkbox "Show Develop menu in menu bar"
```

Comment:
We probably should fix the invalid order here

Reply:
You mean the numbering? Or is something else wrong?

Reply:
just the numbering

Reply:
@jamonholmgren pointed out in https://github.com/react-native-webview/react-native-webview/pull/3081#discussion_r1282326026 that Markdown will still render the numbering correctly, so while it looks weird in the raw view, I guess it doesn't actually hurt the user-facing documentation